### ISSUE TITLE
How to perform Label Encoding ?

### ISSUE THREAD

**Issue by ksachdeva** [tensorflow/tfjs-core#441]
Hi,

Apologies as it is more of a question than an issue. I could not find mailinglist, stackoverflow etc to ask the question and hence posting it here. Please guide if there is a better place to ask the question.

I have a data set and the example is written using python where the author is using the LabelEncoder to transform the name of countries in to numerical data set.

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html

How would we do this using deeplearnjs ?

Regards & thanks
Kapil

**Comment by nsthorat** [comment]
We have [tf.oneHot](https://js.tensorflow.org/api/0.6.1/#oneHot) and [embeddings](https://js.tensorflow.org/api/0.6.1/#layers.embedding) in the higher-level layers API now:
